server/seed.ts

```diff
@@ -1,21 +1,23 @@
 import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
-import Logger from "./logger";
+import { Logger } from "@nestjs/common";
 import { NestExpressApplication } from "@nestjs/platform-express";
 import { EmailService } from "./email/email.service";
 import { ConfigService } from "@nestjs/config";
 import { UsersService } from "./users/users.service";
 import { UtilService } from "./util";
+import loadConfig from "./configLoader";
 
-const initApp = async (options) => {
-    options.logger.log("info", `Loading AppModule`);
+async function initApp() {
+    const defaultConfigFilePath = "config.seed.yaml";
```

Comment:
suggestion: what do you think about moving `defaultConfigFilePath` to the loadConfig function?